keras_cv/layers/preprocessing/random_translation.py

```diff
@@ -232,11 +238,73 @@ def get_random_transformation(self, image=None, **kwargs):
     def _batch_augment(self, inputs):
         # Change to vectorized_map for better performance, as well as work
         # around issue for different tensorspec between inputs and outputs.
-        return tf.vectorized_map(self._augment, inputs)
+        return tf.map_fn(
+            self._augment,
+            inputs,
+            fn_output_signature={
```

Comment:
can you make use of the abstractions for output signatures I have in BaseImageAugmentation layer?  Its actually very hard to hit all of the right cases:

https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/base_image_augmentation_layer.py#L202

Reply:
you can see: https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/base_image_augmentation_layer.py#L240

should handle most cases